src/auth.cpp

```diff
@@ -10,6 +10,17 @@
 namespace b64 = boost::beast::detail::base64;
 using boost::property_tree::ptree;
 
+static void dump_resp_error(string message, gint64 code, ptree resp)
+{
+	cerr << message << ": HTTP_" << code << endl;
+	if (resp.data().length())
+		cerr << resp.data() << endl;
+
+	for (auto it: resp)
```

Comment:
Just really nitpicking :). This loop would be slightly better if defined as `for (const auto& it: resp)`.

Reply:
Updated the code accordingly